### ISSUE TITLE
Understanding doc contradicts SC2.4.5 Multiple ways

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1077]
The second paragraph of the Understanding document says "For a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page where the links on the home page can also serve as a site map." The way I read this is that in this case each page will have only one way to locate it, which directly contradicts the SC, unless I'm reading the paragraph wrong. What does everybody think?

And yes, I agree this might seem a trivial case to raise, but it sets a precedent for ignoring the SC. Suppose you have a 20 page site, but it is the same simple structure as the example in the Understanding, i.e. all 20 pages can be reached from a link somewhere on the Home page and all have a link back to the Home page.

While seemingly trivial, this question takes on more meaning when you consider the tablet and mobile phone responsive layouts. Even a 4 option menu can become a drop down hamburger menu in the phone layout, and we all know how prone they are to not work for all kinds of users. (And the reason I am raising this query is I just had a case of exactly that. It was 4 pages and, yes, the phone version and 400% zoom layouts did not work for keyboard users. If there had been a second menu in the page footer that wrapped lines instead of turning into a drop down menu on the zoomed layout, the site would still have been usable by those people. As it was, it wasn't.) I agree the prime remedy in that case is to fix that failure, but it does highlight that this issue of multiple ways for small sites is not as trivial as it first looks. 

So, should there be this apparent contradiction in the Understanding document?

**Comment by patrickhlauke** [comment]
> The way I read this is that in this case each page will have only one way to locate it, which directly contradicts the SC, unless I'm reading the paragraph wrong. What does everybody think?

I'd agree that that particular paragraph does contradict the SC.

But it brings me to a different/related question: if the paragraph instead said "and on each page there is always link to any other page" (i.e. there's a standard navigation, and every page can be reached from any other page), does that pass? Because if you have N page, there are N-1 different ways to reach that page (from any of the other pages).

The way I've interpreted (loosely) the SC is to mainly be "user doesn't have to remember 'oh, i can only reach THAT page if i'm on THAT OTHER page and there's a link there to it". that basically they can always work out how to reach a page easily, and they don't have to start remembering complex paths/journeys that are the only way to get to a particular page they had in mind.

**Comment by detlevhfischer** [comment]
I think the inaccessibility of the menu button in your example is a different issue covered by 2.1.1 (possibly also 1.3.1 / 4.1.2) and should not enter the discussion of 2.4.5.
I personally think the exception for small sites makes sense. Even if someone links 20 pages directly from the start page and you can go back there and then go to another one of those 20 pages, is that really an issue? It may be unusual or awkward but the simplicty of the structure still seems to suggest you would not need multiple ways to get there?

**Comment by patrickhlauke** [comment]
Dredging this up again, but as I've just had a discussion on this topic...this is one of those SCs that seems slightly "handwavy" in one sense, but then also oddly restrictive.

*IF* we assume that the example with the hairdresser site only passes because it has the following two ways to navigate:

* sequential links from one page to the other (I mean, which site does that, these days?)
* homepage with links to all pages

then are we saying that if every page has a complete navigation on each page, it still *must* have at least either a search or a sitemap? those seem to be the only options possible. and if so, wouldn't the vast majority of the web really be failing this? i rarely find sites with a complete sitemap (that covers absolutely every page), and while it's easy to say that a site can just add a search function (and one that covers pages, not just products or similar), this is not always easy/possible to do on a technical level.

You could even go further and start dissecting what exactly a "set of web pages" is. Is a whole site, by virtue of being a single site (with same/similar look and feel, navigation, footer) a "set of web pages" necessarily? Or is a set more a self-contained section within that site (like an article split over two/three pages)?

Lastly, can this criterion even be applied to something that is app-like? i'd argue that in most cases, it can't. you couldn't have a "map" that contains links/buttons to all possible screens within the app, nor could you create a search of sorts that surfaces all possible screens of an app.

In short, I admit that this is one of those SCs that is very difficult to judge. Either you take it very loosely as meaning "there must be ways for users to reach a page, rather than having to remember exactly one specific path they need to go down/pages they need to traverse in order to reach this page", in which case most times you'd pass. Or you take it very restrictively as essentially mandating a search function (that covers all pages, including static pages, not just a product search or similar on an ecommerce site) or a complete sitemap, which is a non-inconsiderable amount of work for a small-to-medium site.

**Comment by patrickhlauke** [comment]
while on the topic of 2.4.5 ... the advisory technique to use `<link rel="...">`  should probably be retired now too. no browser exposes this in any meaningful way, to my knowledge (yes, old Opera used to...10 years ago or so). not aware of any common/readily available extensions for browsers that leverage these links either.

**Comment by mraccess77** [comment]
Basically most just treat this SC that there are 2 ways to reach the page without specific pathing.  That is there are two other pages that will link to the page - I also agree with you that how you define a set of of pages can make a difference.

**Comment by joelanman** [comment]
I was just auditing a site and was also confused by 2.4.5 and smaller sites. If you have a small site and only link to each page once from the home page, with each page also linking back to the home page, does that fail 2.4.5? The text quoted in the original post seems to say not, but I don't follow the reasoning

>For a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page where the links on the home page can also serve as a site map.

If the home page also serves as a site map.. that's only 1 way of navigating to the pages, not multiple.

To be clear, I'm not sure that a small site with navigation as described is an accessibility problem. So possibly a small site should be listed as an exception to 2.4.5?

**Comment by patrickhlauke** [comment]
> a small site should be listed as an exception to 2.4.5

but then, to be "testable", the working group would have to normatively define what does and doesn't count as a "small site". none of the "3 or 4 pages...may be sufficient...". needs an (arbitrary) number of pages, a threshold where any site with less than or equal to that number of pages (what's a "page"?) is considered "small", and anything above is considered "large" ...

to be clear, I think that'd be fool's gold to do

**Comment by yatil** [comment]
7 – That's the magic number.

**Comment by patrickhlauke** [comment]
coming back to this almost 3 years later, and I'll readily admit that 2.4.5 is an SC that I (and, most accessibility practitioners I've spoken to) mostly reinterpret in a much looser way than what the understanding document hints at.

* it's mostly inapplicable to "application" type sites
* on its face, it's far too onerous and restrictive (currently demanding that every website must have two navigation mechanisms, essentially ... regardless of size of the site, and any associated technical challenges - setting up your own custom search, or creating and then maintaining a complete site map, in addition to the regular navigation)
* there are better search engines out there that can do a much better job at searching for a specific page on a site, without the need for the site itself to build its own search

I admit that I *am* strongly of the opinion that this is an SC whose technical burden would, to me, make it more of a triple-A criterion, and because of its inapplicability in many scenarios, I'm very tempted to say it might be something worth reconsidering/removing...

**Comment by scottaohara** [comment]
for a lot of sites though, what is the actual practical benefit of telling people the must have a search feature, or they must have a site map?

take my website for example https://scottohara.me

with a strict reading of this rule, wouldn't my website fail this rule because i only have a single navigation persistent across all my pages?  Would I not fail this rule because all my blog posts are only available from my writings page?

if the answer is yes - then ok. i can add a site map to re-list all my primary pages - which are already accessible on every page of my website.  and it can list all my blog posts.  so then someone can go to my persistently available writings page to reach blog posts... or the sitemap to reach blog posts.  

if that's what the rule is saying, then ok.  but it seems more like busy work to fail me for than actually making my content actually easier to find.

**Comment by giacomo-petri** [comment]
I agree with @scottaohara and IMO we should be closing this issue, since the exception seems reasonable per Scott's feedback.

**Comment by patrickhlauke** [comment]
We should probably consider adding some kind of clarification/strengthening of the nuance required to evaluate 2.4.5 in the real world...

**Comment by TestPartners** [comment]
I agree that we need clarification. I recently told a client they needed to add a site map page to a five-page website. The website will never change, so it was easier to add a site map than it was to argue about whether it is really needed. This morning, one of my team asked what he should recommend for a three-page website. Some testers may never encounter tiny websites like this, but we test a surprising number of them.

That said, I think that unambiguous clarification will be difficult. The phrase "may be sufficient" on the Understanding page is very unhelpful because it does not mention any criteria by which sufficiency can be assessed.

**Comment by guyhickling** [comment]
I think there is one good consequence from this SC, in spite of its faults. I believe that the SC has very largely been responsible for most sites having a site map. I don’t think they would be anything like as common if the SC did not exist! Even though they are often incomplete, they do at least provide an alternative navigation for the vast numbers of sites that have a main menu that does not work from the keyboard.

**Comment by guyhickling** [comment]
It seems to me that two important things to do now are:
a) Give the SC a rewrite or reconsideration for WCAG3. 
b) And for WCAG2, amend the Understanding to at least give some sort of definitive support for people who are already relying on the Understanding’s apparent "exception" of 3 or 4 page sites. 

For (b), we could consider adding a formal Note to the SC (the best solution, I think), to formally recognise this long standing half-exception. Since the Understanding doc already uses the cut-off point as 3 or 4 pages, we could say, “Websites of four pages or less are excepted providing each page contains a link from and to the Home page.” That will negate the contradiction in the current Understanding document while not changing the sense of the present Understanding by very much.

Alternatively we could slightly amend the second paragraph under the Intent heading in the Understanding document to read “We [or put W3C, or the Working Group here?] consider it acceptable that, for a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page…..” At least this would explain the reason for the contradiction. 

I think definitive support for a specified cut-off point (even if long overdue) is particularly important to website owners who are sued in future for non-compliance with their laws, or in danger of that. True, the a lawsuit will be for much more than non-observance of just this one SC, but we don’t want to be adding uncertainty of the WCAG’s making to the other challenges being made!

**Comment by GreggVan** [comment]
All one needs to do to  meet this SC is to provide a search function on the site.  That counts as a second way to navigate a site.  This was the intent of the Working group when this was written.   If not in the Understanding document that a search function restricted to the site counts as one means of navigation   (and browsing is another)  then it should be added.

**Comment by patrickhlauke** [comment]
> they do at least provide an alternative navigation for the vast numbers of sites that have a main menu that does not work from the keyboard

the site would still fail 2.1.1 Keyboard (unless they made an argument that BECAUSE a keyboard user can use the site map, they don't need to make their actual navigation work with a keyboard...which would be worse, i'd say)

> All one needs to do to meet this SC is to provide a search function on the site.

while i can see this sort of requirement make sense for large content sites, it really does feel like absolute overkill for a small website. and technically it may not be as straightforward anymore (particularly with big search providers limiting the ability to have site-specific searches these days, since they're now mostly intent on pushing their AI slop and not have people even leave their actual search site). and as these days i believe google is the only one that still offers a way to create a custom site search (since the search landscape has changed since this SC was first devised), and not every site is able to build their own search from scratch ... it feels a bit reductive to basically suggest "all sites should just cram google custom search into their code"...

the whole angle of this SC is just ... bizarrely outdated in my view.

**Comment by GreggVan** [comment]
@patrickhlauke 

+1 to your comment about main menus's not being keyboard accessible


RE your second comment on overkill
 --  yes it might be.   But it is an option.   
As is a site map.    
As is a menu that includes all pages (essentially a site map) 

Authors can choose any and all of these.

**Comment by patrickhlauke** [comment]
> As is a menu that includes all pages (essentially a site map)

but based on the logic of the SC, that navigation menu counts as only *one* way. so that's not enough. you're then just left with providing *another* site map or a search engine...

**Comment by patrickhlauke** [comment]
unless we're ok with the stretched logic that if the navigation menu (that includes links to all pages) is present on every N page, in essence every page can be reached by N-1 links (from each of the other pages)

**Comment by TestPartners** [comment]
> unless we're ok with the stretched logic that if the navigation menu (that includes links to all pages) is present on every N page, in essence every page can be reached by N-1 links (from each of the other pages)

That's not my understanding of what the SC is asking for. It's asking for two different mechanisms in case a user has a difficulty (perhaps cognitive) with one mechanism. For instance, someone might not understand how dropdown menus work, so they would benefit from having the same links on a site map page. In user testing on a large website with perfectly constructed primary, secondary and tertiary menus, I observed a screen reader user using the search feature because it was much more efficient for him.

**Comment by patrickhlauke** [comment]
well, that's the crux of it: are we saying that every single website out there which does not have their navigation plus at least either a search or sitemap is non-compliant? this would likely make the vast majority of sites out there fail WCAG. if that is truly the case, that's a very wide overreach for a AA SC...

**Comment by patrickhlauke** [comment]
also, here's a fun extra wrinkle. say a large site like https://www.amazon.co.uk/ ... they do have a search. but the search is specifically for products in their catalogue. their search doesn't surface any of their actual static content pages (say, i want to find the "accessibility" page ... searching for that term brings me products, not static pages on the amazon site)

**Comment by patrickhlauke** [comment]
if the requirement is basically "every site must have two complementary methods to navigate it", then again I'd posit that that's a far more wide-reaching SC than it should be at AA

**Comment by GreggVan** [comment]
browsing is a second way as long as there is a path of some type to all pages from home page OTHER than the nav menu.  but yes - if there is not - then browsing cannot be counted separately from Browsing

**Comment by GreggVan** [comment]
@TestPartners 

the nav menu does not have to be on every page to count.  only the home page  as long as there is a  HOME button on every page

**Comment by TestPartners** [comment]
> [@TestPartners](https://github.com/TestPartners)
> 
> the nav menu does not have to be on every page to count. only the home page as long as there is a HOME button on every page

Indeed. The five-page website I mentioned above was such a design. The home page contained links to every other page, albeit the links were in tiles that also contained plain text, and there wasn't a menu. Every other page only contained a link back to the home page.

**Comment by TestPartners** [comment]
> well, that's the crux of it: are we saying that every single website out there which does not have their navigation plus at least either a search or sitemap is non-compliant? this would likely make the vast majority of sites out there fail WCAG. if that is truly the case, that's a very wide overreach for a AA SC...

The Understanding page also allows one mechanism to be Next and Previous links that navigate linearly through all the pages. This could be supplemented by a conventional navigation menu, in which case a site map or search feature would not be required. This would only be viable on small websites and may not make any sense in some cases.

**Comment by patrickhlauke** [comment]
> The Understanding page also allows one mechanism to be Next and Previous links

oh wow yes, so many modern websites these days have a prev/next link that takes you on a linear tour through an entire site...

**Comment by TestPartners** [comment]
> > The Understanding page also allows one mechanism to be Next and Previous links
> 
> oh wow yes, so many modern websites these days have a prev/next link that takes you on a linear tour through an entire site...

Hey, don't blame the messenger - it's one of the examples on the Understanding page! But for a small website, it might be a simpler solution than implementing a search feature or a site map.

**Comment by patrickhlauke** [comment]
so taking an example (with apologies to @smhigley ... it's just because i had her site open in a tab just now) https://sarahmhigley.com/ ... this site would fail 2.4.5, and by extension not be WCAG 2.x compliant, right? because despite being relatively small, and logically structured, it's not also implemented a site map and search?

**Comment by TestPartners** [comment]
According to both the normative text and the Understanding page, yes, it's non-conformant because there is only one means of locating a page. This is an objective fact, not just my opinion. I doubt if it significantly affects the accessibility, but there is no exception for that. You can say that the non-conformance doesn't matter, but you can't say the website is conformant.

It would be interesting to hear from the people who created this SC nearly 20 years ago. They must have had this sort of discussion back then. What research did they rely on when drafting this SC?

**Comment by patrickhlauke** [comment]
> What research did they rely on when drafting this SC?

I suspect somebody was adamant at the time that "I need to be able to do this!" and shouted long enough for it to be enshrined into an SC. Then the artificial bogus example of "well, if it's a small site, maybe it's enough to do this" was added later when it was realised that the SC was far too prescriptive/restrictive...

**Comment by TestPartners** [comment]
Well, we're stuck with it for WCAG 2.x, so all we can do is improve the guidance if that's possible. I don't even know how to do that, given how prescriptive the normative text is. I suspect that a lot of testers ignore this SC when testing small websites, even though they shouldn't, especially if they have bigger battles to fight.

**Comment by patrickhlauke** [comment]
Could fight to deprecate it, though I suspect the appetite for that is quite low... or expand the understanding to add much more lenient parameters, though that should arguably not be done in non-normative understanding docs anyway...

**Comment by guyhickling** [comment]
How about the two suggestions above? I'll repeat them here for ease of reference:

....... we could consider adding a formal Note to the SC (the best solution, I think), to formally recognise this long standing half-exception. Since the Understanding doc already uses the cut-off point of 3 or 4 pages, we could say, “Websites of four pages or less are excepted providing each page contains a link from and to the Home page.” That will negate the contradiction in the current Understanding document while not changing the sense of the present Understanding by very much.

Alternatively we could slightly amend the second paragraph under the Intent heading in the Understanding document to read “We [or put W3C, or the Working Group here?] consider it acceptable that, for a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page…..” At least this would explain the reason for the contradiction.

I agree that any cut-off point is arbitrary - we could argue making it 5 pages, or even 10 pages. But at least having a formal exception for the very smallest sites will provide some semblance of sense to this SC.

**Comment by patrickhlauke** [comment]
Adding a formal note / other amendment to the normative wording of the SC would also work, but just like deprecation (which would be my preference, rather than adding some arbitrary guardrails), this is again a normative change which the wider group may just not have the desire to engage with.

tweaking the understanding to explain more clearly that "if the homepage - or some other page - already acts effectively as the sitemap, linking to each page, AND there is some other navigation mechanism, then that's fine" might work to clarify things...but i still think this is a weirdly spurious and artificial example. would the homepage have links to all pages and NO general navigation (because i doubt anybody would set up a site with a navigation and then also still link to every page from the main content of the homepage)? the navigation only appears once you're one level deep? or are we saying that the homepage having the general navigation that already links to all pages DOES weirdly make the homepage a sitemap as such (but then, every other page that has that navigation would also be a sitemap). it's just...weirdly contrived. and still doesn't seem to actually hint at an exemption as such.

**Comment by patrickhlauke** [comment]
I'll also add, as an interesting point: while Google still does allow you to just hook in a custom site search, it will also push ads as part of it. If you don't want that for your site, you need to pay Google for the privilege.

> Ads
> Nonprofit organisations, accredited educational institutions and government agencies may qualify for a free version of Programmable Search Engine without ads. [Learn more](https://support.google.com/programmable-search/answer/12423873)
> Advertising can be disabled by paying per search query instead. Get started with the [Programmable Search element-paid API](https://developers.google.com/custom-search/docs/paid_element) to set up billing and generate an API key.
> 
> Enter a Programmable Search element-paid API key to enable the feature, or leave empty to disable the feature.
> Paid element API key
> Add API key

So even for search, if you as a site owner haven't got the technical knowledge/infrastructure to set up your own search, you either have to pay, or accept that your site starts hosting ads... seems a bit extortionate.

"Or make a site map" - if it's a blog site for instance, you'd need to make this dynamically update to list every. single. blog post, i'd assume.

**Comment by patrickhlauke** [comment]
Maybe the cheapest option: include a link called "Search" on your site that simply goes to one of the existing large search engines, with a `site:` parameter already set (e.g. https://www.google.com/search?q=site%3Awww.splintered.co.uk)

**Comment by guyhickling** [comment]
> i doubt anybody would set up a site with a navigation and then also still link to every page from the main content of the homepage

I think you misunderstand me. All I'm saying is that on very small sites of just a few pages, having a main menu should be sufficient (with links back to the home page as well, of course). Even small sites usually have a main menu (but, yes, if a site has just a few links on the home page and no menu that would be ok as well for the few sites that go down that route). 

All I'm suggesting is that we should regularise that very simple approach of small sites in a formal way, whether by using a formal Note or my clarifying it in the Understanding doc. Just to lessen the confusion.

**Comment by TestPartners** [comment]
While I agree in principle, this sort of discussion usually gets bogged down in arguments over definitions, such as what constitutes a small website. Four pages is certainly small. Ten might be. Twenty? We deal with subjective judgements like this all the time in everyday life, yet we struggle with them when writing specifications and rules.

**Comment by patrickhlauke** [comment]
I think part of the whole confusion here is that weird theoretical example the understanding provides in such a handwavy way:

> For a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page where the links on the home page can also serve as a site map.

There patently are NOT at least two ways to reach each page, or am I missing something? If this is indeed one homepage with links to all other pages (which may be in the navigation?), and the child pages all just have (at least) a link back to the homepage (but they could also still have the full navigation to all other pages), how can they count as having two ways? And does this try to define an exception when it clearly isn't the job of the understanding docs to do that?

Also...even if we accepted it... "it may be sufficient" adds even more handwaving on top. Is it sufficient or isn't it? This is just bad wording all around.

**Comment by TestPartners** [comment]
Agreed. The Understanding page is far more coherent if you remove that example. Doing so doesn't address the fact that the SC is arguably too onerous for level AA, but at least it would enable us to apply the SC consistently.

**Comment by guyhickling** [comment]
I think we may be reading the Understanding doc in different ways. Some comments above seem to be interpreting the Understanding paragraph as a clumsy attempt to rationalise a concept that having links back to the home page, which then has the menu to get to all the other pages, somehow constitutes a ‘second’ way to get to all pages. Which as Patrick has just pointed out, is clearly not the case.

Whereas, I always thought that paragraph in the Understanding was a clumsy attempt to make an exception for small sites. Which is clearly the wrong way to do that, of course, as any exception ought to be in the SC itself, but that is what I thought it meant! But on that basis I have never faulted a small website in my audits, that only had a main menu and no other alternative.

That is why I have been arguing to add a Note to make a formal exception for small sites of 4 pages or less, since the general idea of that exception seemed to me to be already in the understanding doc, so such a Note would not be proposing something entirely new. And on that basis I would not remove the paragraph in question.

**Comment by mraccess77** [comment]
I've interpreted it to mean the homepage acts as a sitemap page.  if the homepage link is on every page.  While generally, two links are needed to reach each page- in this case the home page is doing double duty and adding a sitemap link to each page wouldn't provide any additional benefit given the site structure and thus in this case would be sufficient.  Most other sites don't have a home page that acts like a sitemap and thus something like a sitemap or search or something else which allows two ways of access would be required.

**Comment by TestPartners** [comment]
That's a sensible interpretation, although it does not conform with the normative text. Leaving that aside, it also raises a question regarding the example on the Understanding page. If we accept your interpretation, a home page that contains links to every other page would always be sufficient. But the example says "may be sufficient", which implies that it may not be sufficient. Can we think of any situation in which it would not be sufficient? If so, we should add it to the Understanding page. If not, perhaps we should change the Understanding page to say the technique "is sufficient".

However, that opens a potential loophole. I recently tested a website that contained the craziest mega menu I have ever seen. The top level menu contained about ten links. Each link opened a huge mega menu that contained a tabbed interface with up to ten tabs.  Each tabpanel contained up to ten links, and in total the menu contained around one thousand links. If the menu contains links to every page, which it may well do, this is sufficient to meet the SC, according to your interpretation.

Defining a "small" website as being one in which the home page contains links to every other page has the advantage that it avoids specifying the threshold in terms of the number of pages. But it allows a 1,000-page website to meet the definition of "small" if it has a crazy menu as described above. I do not regard it as small and I would require it to have a search feature or site map.

**Comment by guyhickling** [comment]
Thank you Jonathan, that is clear. If not even the experts (if I may call us that for the moment!) read it the same way, it doesn’t give much hope to everybody else!

Just one thing. We cannot simply delete this paragraph, because that would remove the “exemption” or “rationalisation” for small sites (no matter how clumsy it was worded), and that in turn would place thousands of websites in non-compliance that were previously validated by auditors and testers as being compliant. At very least, we need to decide on a workable meaning and have the paragraph edited to say that more clearly.

**Comment by patrickhlauke** [comment]
> We cannot simply delete this paragraph, because that would remove the “exemption” or “rationalisation” for small sites

that exemption never existed as such. it's a (badly) worded example in a non-normative document. a real exemption would need to be in normative text.

while non-normative text can clarify the normative wording (along the lines of "when we say X, we mean Y only, and not also Z") which can limit the scope of normative wording a bit (or rather, clarify what the intended scope is/was), it can't just magic up actual hard exemptions (and, even if in this case it tried to do it, it didn't even manage that, using "should be sufficient")

if the paragraph was intended to serve as an explanation of how that (fictitious) small site actually did have two distinct "ways" to reach pages (which is still not clear), it needs to be reworded to actually make sense. but then, IF that rationale works for "small" sites, why wouldn't the rationale work for "medium" or "large" sites too? just very clumsy and contradictory either way...

**Comment by guyhickling** [comment]
> that exemption never existed as such. it's a (badly) worded example in a non-normative document

Technically, I agree with you of course. Nevertheless I'm sure lots of auditors and testers have passed small sites on that basis even though there was not a second way to access the pages. It has been a 'de facto' exemption, and I believe we have to accept it now, not suddenly turn round and say all those sites are now failing the SC. It would also put all those auditors in a difficult position if they have to re-test such a site. 

And we here would be insisting on scrupulous observance of an SC that even we have serious doubts about!

**Comment by TestPartners** [comment]
I don't accept that we have to just accept the way things have always been done (by some people) if it is demonstrably wrong. Just look at the (interminable) discussion regarding audio description. A great many people, including me and my team, used an interpretation that we recently found is not what the authors intended. We didn't claim to have created a de facto interpretation, but we changed our interpretation even though that meant telling clients we had erroneously reported their videos as non-conforming. We need to do the same sort of thing with this SC.

**Comment by GreggVan** [comment]
The end result is that -- if a site does not pass- they can create a way to pass easily.

one option is to just put a button the triggers a search with   site:<theirSite>   in the  search string.

**Comment by patrickhlauke** [comment]
> one option is to just put a button the triggers a search with site: in the search string.

as per https://github.com/w3c/wcag/issues/1077#issuecomment-3249472489

it does make me wonder though...for users that do actually have a problem with navigating a site...would they not already know how to do this themselves?
